src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -68,15 +87,15 @@ public void runClimber() {
     }
 
     public void raiseRobot() {
-        climber.set(TalonFXControlMode.PercentOutput, 1.0);
+        climber.set(TalonFXControlMode.Velocity, climberRpms);
     }
 
     public void lowerRobot() {
-        climber.set(TalonFXControlMode.PercentOutput, -1.0);
+        climber.set(TalonFXControlMode.Velocity, -climberRpms);
     }
 
     public void holdRobot() {
-        climber.set(TalonFXControlMode.PercentOutput, -0.0);
+        climber.set(TalonFXControlMode.Velocity, 0.0);
```

Comment:
We could hold in position or velocity mode, not sure which is better

Reply:
it is easier if everything is in velocity so for now I say we see how velocity works

Reply:
👍